tests/src/JIT/Performance/CodeQuality/Inlining/InlineGCStruct.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using System;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+[assembly: OptimizeForBenchmarks]
+[assembly: MeasureInstructionsRetired]
+
+public class InlineGCStruct
+{
+
+#if DEBUG
+    public const int Iterations = 1;
+#else
+    public const int Iterations = 2500000;
+#endif
+
+[MethodImpl(MethodImplOptions.NoInlining)]
+public static int FastFunctionNotCallingStringFormat(int param)
+{
```

Comment:
Could you add a comment that explain what the purpose of the test is and that we expect that the method System.Format is not inlined here.  WE may run this test case using JIT32 or JIT64 and if it fails its nice to have a comment explaining what is expected behavior.

You could also reference this PR is the comment as well https://github.com/dotnet/coreclr/pull/7727